Dockerfile

```diff
@@ -15,6 +15,10 @@ RUN mix local.hex --force && \
 
 RUN apk add --update nodejs nodejs-npm
 
+ARG SENTRY_ORG=$SENTRY_ORG
+ARG SENTRY_PROJECT=$SENTRY_PROJECT
+ARG SENTRY_AUTH_TOKEN=$SENTRY_AUTH_TOKEN
```

Comment:
Where is this defined, what's the path it takes to get here, and does it need to be secret?

I'm not familiar with `ARG` (or with docker at all really), but in trying to figure out what this does, I saw this:
> ARG values can be easily inspected after an image is built, by viewing the docker history of an image. Thus they are a poor choice for sensitive data.
https://vsupalov.com/docker-arg-env-variable-guide/

Is that an issue for us?

Reply:
These start off as the corresponding `SENTRY_*` env vars in dev. The `--build-arg` options added to `build.sh` expose those for use inside the `Dockerfile`, and calling `ARG` makes those environment variables available during the build, but not to the resulting image itself, which is exactly what we want here.

I don't think we have to worry about anyone viewing the image history, because IIUC if someone has access to that, they probably have wide-ranging access to our server in general, perhaps root access, so we already have bigger problems.